Ladybird/AppKit/Application/EventLoopImplementation.mm

```diff
@@ -82,12 +82,12 @@ static void post_application_event()
     return timer_id;
 }
 
-void CFEventLoopManager::unregister_timer(int timer_id)
+void CFEventLoopManager::unregister_timer(intptr_t timer_id)
 {
     auto& thread_data = ThreadData::the();
-    thread_data.timer_id_allocator.deallocate(timer_id);
+    thread_data.timer_id_allocator.deallocate(static_cast<int>(timer_id));
 
-    auto timer = thread_data.timers.take(timer_id);
+    auto timer = thread_data.timers.take(static_cast<int>(timer_id));
```

Comment:
eh. Should we make the timer_id_allocator use the type that the timer_id actually is? This will truncate timer addresses to 32-bits on an LP64 system, which seems like it could theoretically collide.

Reply:
> This will truncate timer addresses to 32-bits on an LP64 system.

Not really. MacOS event loop still uses IDAllocator, so these `intptr_t`s are just `int`s in disguise.

Reply:
aha I see. makes sense then.